eli5/xgboost.py

```diff
@@ -91,7 +93,9 @@ def explain_prediction_xgboost(
         target_names=None,
         targets=None,
         feature_names=None,
-        vectorized=False):
+        vectorized=False,
+        expand_missing_features=False,
```

Comment:
We'll also need an ability to provide an inverse of feature_re, or make it accept a function - else we won't be able to filter out missing features, right?
I like an option to show combined weights for filtered out features, it can be useful for all classifiers, not just for xgboost.